common-setup.sh

```diff
@@ -133,17 +141,17 @@ EOF
     fi
 }
 
-allow_broken_macos="$HOME/.local/share/reflex-platform/allow-broken-macos"
-if (sw_vers | grep "ProductVersion" | grep "\(10.13.0\|10.13.2\)$" || false)  &> /dev/null; then
+if on_darwin_host && version_greater_than "10.13.2" "$(sw_vers | grep "ProductVersion" | cut -f 2)"; then
```

Comment:
Won't this block running on Sierra and below ?  
I was expecting a version_in_between thing, e.g. `version_greater_than "10.13.2" x && version_greater_than x "10.13"`

Reply:
Ah, I just assumed the message "prior to" meant all macOS versions!

Reply:
I'll do the other half of the comparison but now this complexity is questionably worthwhile...

Reply:
`*High Sierra* versions prior to 10.13.2`. 
I admit, the phrasing isn't the best, but everything short of listing them sounds awkward to me. 

> I'll do the other half of the comparison but now this complexity is questionably worthwhile...
Yeah, that happens a lot while working on the shell stuff